### PR TITLE
[native] Add e2e test for timezone_hour and timezone_minute

### PR DESCRIPTION
**Pull request by gopukrishnasIBM**
Test plan - Added e2e tests for new scalar function timezone_hour.
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

**Comment by yzhang1991** [comment]
Hi @gopukrishnasIBM can you follow up on this and get it ready for review? Thanks!
CC @ashokku2022

File: presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java [hunk 1]
```diff
@@ -857,6 +857,9 @@ public void testTimestampWithTimeZone()
 
         assertQuery("SELECT orderkey, year(from_unixtime(orderkey, '+01:00')), quarter(from_unixtime(orderkey, '-07:00')), month(from_unixtime(orderkey, '+00:00')), day(from_unixtime(orderkey, '-13:00')), day_of_week(from_unixtime(orderkey, '+03:00')), day_of_year(from_unixtime(orderkey, '-13:00')), year_of_week(from_unixtime(orderkey, '+14:00')), hour(from_unixtime(orderkey, '+01:00')), minute(from_unixtime(orderkey, '+01:00')), second(from_unixtime(orderkey, '-07:00')), millisecond(from_unixtime(orderkey, '+03:00')) FROM orders");
         assertQuery("SELECT orderkey, date_trunc('year', from_unixtime(orderkey, '-03:00')), date_trunc('quarter', from_unixtime(orderkey, '+14:00')), date_trunc('month', from_unixtime(orderkey, '+03:00')), date_trunc('day', from_unixtime(orderkey, '-07:00')), date_trunc('hour', from_unixtime(orderkey, '-09:30')), date_trunc('minute', from_unixtime(orderkey, '+05:30')), date_trunc('second', from_unixtime(orderkey, '+00:00')) FROM orders");
+        
+        assertQuery("SELECT timezone_hour(from_unixtime(orderkey, 'Asia/Oral')) FROM ORDERS");
```
Comment (aditi-pandit) [inline presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:861]: Table name "ORDERS" should be in lowercase (for both this line and the next)

Comment (gopukrishnasIBM) [reply presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java]: @aditi-pandit Changed ORDERS to orders

**Review by aditi-pandit — APPROVED** [review]

**Comment by yzhang1991** [comment]
Hi @gopukrishnasIBM, it seems like there is a failing test:
```
[INFO] There is 1 error reported by Checkstyle 8.16 with src/checkstyle/presto-checks.xml ruleset.
Error:  src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:[1204] (regexp) RegexpMultiline: Line has trailing whitespace
```

Can you make a quick update?

**Comment by gopukrishnasIBM** [comment]
> Hi @gopukrishnasIBM, it seems like there is a failing test:
> 
> ```
> [INFO] There is 1 error reported by Checkstyle 8.16 with src/checkstyle/presto-checks.xml ruleset.
> Error:  src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:[1204] (regexp) RegexpMultiline: Line has trailing whitespace
> ```
> 
> Can you make a quick update?

Removed trailing whitespaces.

**Comment by aditi-pandit** [comment]
@gopukrishnasIBM : Please can you fix the CLA for the check.

**Comment by gopukrishnasIBM** [comment]
@aditi-pandit CLA is fixed.
@yzhang1991 Rebased with master.

**Comment by yzhang1991** [comment]
Hi @gopukrishnasIBM please fix the failed test
```
Error:  src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java:[1209,1] (whitespace) FileTabCharacter: File contains tab characters (this is the first instance).
```

**Pull request merged**